Library/Formula/rpm.rb

```diff
@@ -52,18 +56,20 @@ def install
         --with-neon=external
         --with-uuid=external
         --with-pcre=external
-        --with-lua=internal
-        --with-syck=internal
+        --with-lua=external
+        --with-syck=external
```

Comment:
I disagree, I think we should use internal dependencies unless there's a reason to use external ones, thanks.